cap_anndata/cap_anndata.py

```diff
@@ -94,6 +94,22 @@ def _read_df(self, key: str, columns: List[str]) -> CapAnnDataDF:
     def _write_elem_lzf(self, dest_key: str, elem: any) -> None:
         write_elem(self._file, dest_key, elem, dataset_kwargs={"compression": "lzf"})
 
+    def _validate_cap_df(self, cap_df: CapAnnDataDF, axis: int) -> None:
```

Comment:
could be used enum to prevent different values for axis

```
from enum import IntEnum 

class Axis(IntEnum): 
    X = 0
    Y = 1
```

Reply:
We still need to write some sort of validation in this case, because we can't ensure that the function will never get a non-axis value (in general). So, to replace two lines of code, we need to create a new class, add string validation, which looks weird to me. Besides, axis=0 or axis=1 is a common notation. Let's keep it as it is if you don't really mind